programmers/문자열내-p와-y의-갯수.js

```diff
@@ -0,0 +1,16 @@
+function solution(s){
+    var pNum = s.match(/p/gi);
+    var yNum = s.match(/y/gi);
```

Comment:
ES5 부터 var는 사용하지 않음
변하지않는 상수는 const 변하는 값을 가진다면  let 을 선언
```suggestion
    const pNum = s.match(/p/gi);
    const yNum = s.match(/y/gi);
```